roles/kustomize_deploy/tasks/execute_step.yml

```diff
@@ -192,25 +192,29 @@
              path_join | realpath
           }}
       block:
-        # We have to use plain `kustomize build`: lookup would
+        # We have to use plain `oc kustomize`: lookup would
         # be executed locally, on the ansible-controller. In CI,
         # that would be the zuul-executor, and we don't manage them,
         # leading to a risk to either NOT have kustomize, or an old,
         # deprecated version.
         - name: "Build kustomized content for {{ stage.path }}"
-          register: _kustomize_output
           environment:
             PATH: "{{ cifmw_path | default(ansible_env.PATH) }}"
           ansible.builtin.command:
             chdir: "{{ _chdir }}"
-            cmd: kustomize build
+            cmd: "oc kustomize -o {{ _output }}"
```

Comment:
why don't we just capture the output and use the copy to inject it? Everything would be automated, and backup would be in the "right" order/naming... ?